src/components/Details.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import intel from '../assets/intel.png';
+import Agility from '../assets/agile.png';
+import Stren from '../assets/strength.png';
+import { getStrengthHeroesList } from '../Redux/strength/Strength';
+import { getIntelligenceHeroesList } from '../Redux/intelligence/Intelligence';
+import { getAgilityHeroesList } from '../Redux/agility/Agility';
+
+function Details() {
+  const location = useLocation();
+  const data = location.state.att;
+  let heroes = {};
+  let headLine = '';
+  const baseURL = 'https://api.opendota.com';
+  const dispatch = useDispatch();
+  let picture = '';
+  if (data === 'int') {
+    headLine = 'Intelligence';
+    picture = intel;
+    heroes = useSelector((state) => state.intelligenceHeroReducer.heroes);
+    useEffect(() => {
+      dispatch(getIntelligenceHeroesList());
+    }, [dispatch]);
+  }
+  if (data === 'agi') {
+    headLine = 'Agility';
+    picture = Agility;
+    heroes = useSelector((state) => state.agilityHeroReducer.heroes);
+    useEffect(() => {
+      dispatch(getAgilityHeroesList());
+    }, [dispatch]);
+  }
+  if (data === 'str') {
+    headLine = 'Strength';
+    picture = Stren;
+    heroes = useSelector((state) => state.strengthHeroReducer.heroes);
+    useEffect(() => {
+      dispatch(getStrengthHeroesList());
+    }, [dispatch]);
+  }
+  return (
+    <>
+      <div className="bg-[#AD0505] flex gap-6 items-center justify-center">
+        <img src={picture} alt="Dota logo" className="h-36 opacity-95" />
+        <div className="text-white">
+          <p className="font-black text-3xl">{headLine}</p>
```

Comment:
Optional -
Please I strongly suggest that you add some details to the details page to add more depth to your project. 😄